src/components/Contacts/ContactDetails/ContactDetailsTab/People/Items/PersonModal/PersonModal.tsx

```diff
@@ -54,21 +63,24 @@ const LoadingIndicator = styled(CircularProgress)(({ theme }) => ({
   margin: theme.spacing(0, 1, 0, 0),
 }));
 
-interface EditPersonModalProps {
-  person: ContactDetailsTabQuery['contact']['people']['nodes'][0];
+interface PersonModalProps {
+  person?: ContactDetailsTabQuery['contact']['people']['nodes'][0];
+  contactId: string;
   accountListId: string;
   handleClose: () => void;
 }
 
-export const EditPersonModal: React.FC<EditPersonModalProps> = ({
+export const PersonModal: React.FC<PersonModalProps> = ({
   person,
+  contactId,
   accountListId,
   handleClose,
-}): ReactElement<EditPersonModalProps> => {
+}): ReactElement<PersonModalProps> => {
```

Comment:
Doesn't `React.FC` provide this?
```suggestion
}) => {
```

Reply:
Yeah, I'll update this.